etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/sftp/UIFTPUser.java

```diff
@@ -121,7 +126,8 @@ public void showMessage(String message) {
             }
             return response;
         } else {
-            return null;  // cancel
+            return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Return an empty array instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1168)